botorch/acquisition/proximal.py

```diff
@@ -95,10 +95,15 @@ def forward(self, X: Tensor) -> Tensor:
 
         train_inputs = model.train_inputs[0]
 
+        # if the model is ModelListGP then get the first model
         if isinstance(model, ModelListGP):
             train_inputs = train_inputs[0]
             model = model.models[0]
 
+        # if the model has more than one output get the first copy of training inputs
+        if model.num_outputs > 1:
```

Comment:
It doesn't need to be the case that a model with `num_outputs > 1` has copies of the training inputs. Maybe a more precise way of identifying the case you're after here would be to do `if isinstance(model, BatchedMultiOutputGPyTorchModel)`.

Reply:
I added the check for BatchedMultiOutputGPyTorchModel, but I still need the number of outputs to be greater than 1